science/memesuite

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                memesuite
+version             5.1.1
+categories          science biology
+platforms           darwin linux
+license             Noncommercial
+maintainers         uw.edu:cegrant uw.edu:wnoble gmail.com:tlawbailey
```

Comment:
Please add your GitHub handle here too. Have the other people agreed to maintain this port? Also, can `openmaintainer` be added?

Reply:
GitHub handle added, and openmainter. The other maintainers have agreed to maintain this port.